### DISCUSSION TITLE
Aitoc gave up on Magento 1 extensions

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3735] [News]
One of the top developers for Magento has chosen to remove from sale the extensions on version 1 and completely give up support for online stores. I noticed this today, but I think it has been effective for many months.

This is sad news. Those who manage the OpenMage project can discuss with Aitoc to offer those extensions on GitHub as happened with another well-known developer last year. In this way, those who continue the Magento 1 saga through OpenMage can appreciate the past work of some experienced programmers.

**Comment by fballiano** [comment]
I tweeted at them right now, thanks for pointing this out :-)

https://twitter.com/fballiano/status/1747641653341323465

**Reply by addison74** [reply]
Please send them a message here hello@aitoc.com. I think is more efficient to discuss this in private.
